.rubocop_todo.yml

```diff
@@ -0,0 +1,841 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2016-10-13 16:34:45 +0200 using RuboCop version 0.43.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 3
+Lint/AmbiguousRegexpLiteral:
+  Exclude:
+    - 'test/commands/generate/action_test.rb'
+
+# Offense count: 2
+# Configuration parameters: AllowSafeAssignment.
+Lint/AssignmentInCondition:
+  Exclude:
+    - 'lib/hanami/container.rb'
+    - 'lib/hanami/rendering_policy.rb'
+
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: AlignWith, SupportedStyles, AutoCorrect.
+# SupportedStyles: keyword, variable, start_of_line
+Lint/EndAlignment:
+  Exclude:
+    - 'lib/hanami/rendering_policy.rb'
+    - 'test/fixtures/security_headers/apps/web/application.rb'
+
+# Offense count: 1
+Lint/EnsureReturn:
+  Exclude:
+    - 'lib/hanami/commands/console.rb'
+
+# Offense count: 1
+Lint/Eval:
+  Exclude:
+    - 'lib/hanami/config/mapper.rb'
+
+# Offense count: 2
+Lint/HandleExceptions:
+  Exclude:
+    - 'lib/hanami/commands/console.rb'
+    - 'lib/hanami/environment.rb'
+
+# Offense count: 1
+Lint/UnderscorePrefixedVariableName:
+  Exclude:
+    - 'lib/hanami/commands/server.rb'
+
+# Offense count: 46
+# Cop supports --auto-correct.
+# Configuration parameters: IgnoreEmptyBlocks, AllowUnusedKeywordArguments.
+Lint/UnusedBlockArgument:
+  Exclude:
+    - 'lib/hanami/commands/generate/app.rb'
+    - 'test/commands/generate/action_test.rb'
+    - 'test/commands/generate/app_test.rb'
+    - 'test/commands/generate/mailer_test.rb'
+    - 'test/commands/generate/migration_test.rb'
+    - 'test/commands/generate/model_test.rb'
+    - 'test/commands/new/app_test.rb'
+    - 'test/commands/new/container_test.rb'
+    - 'test/fixtures/config.ru'
+
+# Offense count: 40
+# Cop supports --auto-correct.
+# Configuration parameters: AllowUnusedKeywordArguments, IgnoreEmptyMethods.
+Lint/UnusedMethodArgument:
+  Enabled: false
```

Comment:
This is the configuration generated by rubocop. Haven't looked into it, but I was assuming it would generate a more fine-grained list of forgiven offences, rather than disable cops altogether.